lib/format-phone-number-string.ts

```diff
@@ -1,3 +1,3 @@
-export default (phone: number, { prefix = '+' } = {}): string => phone
+export default (phone: number, { prefix = '+', space = ' ' } = {}): string => phone
```

Comment:
I would suggest to change the name of the option from `space`  to `separator`.

Reply:
But it is not the only separator in result of this function. We also have `-` between  last three groups.
So, I think that space is more semantic name for this option.